### PR TITLE
Rails_react Hello World

### PR DESCRIPTION
**Pull request by haliljon**
## In this PR I was able to:

- Implement a connection between a Ruby on Rails back-end and React front-end.
- Understand the pros and cons of different approaches of connecting Ruby on Rails back-end with React front-end.

### CODE REVIEW

File: app/javascript/components/Greetings.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React, { useEffect } from "react";
+import { fetchGreetings } from "../redux_store/greeting";
+import { useDispatch, useSelector } from "react-redux";
+
+function Greetings() {
+    const dispatch = useDispatch();
+    const greetingsData = useSelector((state) => state.greetings.greetings);
+    const singleGreeting = greetingsData[Math.floor(Math.random() * greetingsData.length)]
```
Comment (andersonlebon) [inline app/javascript/components/Greetings.js:8]: - [ ] As I have just said in the above comment the API endpoint should return only one random greeting. Kindly make sure the logic above is implemented in the API contronller.

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @haliljon 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/javascript/components/Greetings.js (hunk 1 above)
Comment (haliljon) [reply app/javascript/components/Greetings.js]: Hi @andersonlebon 

It is done ✅

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @haliljon, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**